tests/testthat/test.fd_inventory.R

```diff
@@ -0,0 +1,19 @@
+context("Loading internal package data")
+
+test_that("Inventory table", {
+  dat <- fd_inventory()
+
+  expect_is(dat$DBH_cm, "numeric")
```

Comment:
Also check that values are positive?

```suggestion
  expect_is(dat$DBH_cm, "numeric")
  expect_true(all(dat$DBH_cm > 0 | is.na(dat$DBH_cm)))
```